README.md

```diff
@@ -1,2 +1,35 @@
 # dockerized-openQA
-Provide a docker image of package openQA
+
+Provide a docker image of package openQA and openQA-worker
+
+
+## How to for collaborators
+
+#### webui
+
+1. Build the docker image
+
+```bash
+# You may need root privileges or your user should belong to docker group.
+docker-build -t openqa-webui:4.5.1509097496.8b8e8e20 github/dockerized-openQA/webui/
+```
+
+2. Start the webui
+
+```bash
+# You may need root privileges or your user should belong to docker group.
+docker run --rm -it --name openqa_webui -p 80:80 -p 873:873 openqa-webui:4.5.1509097496.8b8e8e20 bash
+
+# then
+/root/run_openqa.sh
```

Comment:
You use such way of starting container , because you wanted to debug it . Normally users will not need this . So proper way would be to use command 
`docker run --rm --name openqa_webui -p 80:80 -p 873:873 openqa-webui:latest`